sourcecode/docs/docs.edlib.com/src/pages/contact-us.md

```diff
@@ -4,7 +4,7 @@ title: Contact Us
 
 # Contact Us
 
-Do you need help with your next educational project? We can help with your online courses and educational games. Apart from [Edlib](https://github.com/cerpus/Edlib), we are also building our own educational applications &mdash;[Gamilab](https://gamilab.com/) and [Edstep](https://edstep.com/)&mdash; and we would love to help you build yours.
+Do you need help with your next educational project? We can help with implementing Edlib to your system. Apart from [Edlib](https://github.com/cerpus/Edlib), we are also building our own educational game platform &mdash;[Gamilab](https://gamilab.com/) that utilize Edlib and we would love to help you build yours.
```

Comment:
implementing Edlib to -> embedding Edlib with

utilize -> utilizes

Reply:
Fixed below.